.travis.yml

```diff
@@ -206,19 +218,17 @@ matrix:
       name: 'Frontend [test]'
       env: TEST_SUITE=js
       before_install:
-        - find "$NODE_DIR" -type d -empty -delete
-        - nvm install
+        - *install_node
       install:
-        - ./bin/yarn install --frozen-lockfile
+        - yarn install --frozen-lockfile
```

Comment:
Follow up idea: this command also seems repeated so maybe abstract that out as you did with `*install_node`?

Reply:
Good idea, will do.